gcp/appengine/frontend_handlers.py

```diff
@@ -190,6 +191,39 @@ def add_source_info(bug, response):
   response['source_link'] = response['source']
 
 
+def add_related_aliases(bug: osv.Bug, response):
+  """Add links to other osv entries that's related through aliases"""
+  # Add links to other entires if they exist
+  aliases = dict()
+  for alias in bug.aliases:
+    result = bug.get_by_id(alias)
+    aliases[alias] = {'exists': result, 'same_alias_entries': list()}
+
+  # Add links to other entries that have the same alias
+  if bug.aliases:
+    query = osv.Bug.query(osv.Bug.aliases.IN(bug.aliases))
```

Comment:
Can we just merge this case with the case on line 214 by doing the IN on bug.aliases + [bug.id()] ?